docs/insights/README.md

```diff
@@ -119,3 +119,22 @@ For example, to remove "Unused Parameters" Insight only for some file:
     ],
 ```
 
+### Suppressing insights locally
+
+You could also suppress some insights locally using `@phpcsSuppress` annotation.  
+
+```php
+    /**
+     * @phpcsSuppress SlevomatCodingStandard.Functions.UnusedParameter
+     * @param \Illuminate\Foundation\Auth\User|null $user
+     *
+     * @return bool
+     */
+    public function viewAny(?User $user)
+    {
+        return true;
+    }
+```
+
+You can see [slevomat/coding-standard docs](https://github.com/slevomat/coding-standard#suppressing-sniffs-locally) for more info
```

Comment:
This is a feature specific to `slevomat` insights?

Reply:
I think that it is specific for Slevomat, as it has to know the context to only silence it for that code block.

In phpcs you have the `phpcs:disable` and `phpcs:enable` commands to specify which code blocks you want to ignore, however Slevomat is using the context of the sniff instead to figure out which part you want to ignore. The `phpcs:disable` command also supports specifying a sniff.